### DISCUSSION TITLE
Blog post highlighting keymap editor

### DISCUSSION THREAD

**Discussion by robinovitch61** [nickcoutsos/keymap-editor#152] [Show and tell]
I wrote this that highlights your tool :) love it, thanks so much for making it

https://theleo.zone/posts/configuring-glove80/

**Comment by nickcoutsos** [comment]
Very cool! I'm glad to see that it's been helpful!

Regarding formatting and whitespace, I think you've actually exposed a bug. When saving changes it will reformat the file which in effect parses it into individual syntax nodes and putting them together consistently based on some rules. Normally adjacent nodes (devicetree nodes) should be separated from one another by a blank line, but I'm guessing that somewhere along the way it stopped seeing _labelled_ nodes as being effectively the same type. I'll have to dig in a little deeper to confirm, but if that's the case the fix would be pretty straightforward.

Thanks!

**Reply by nickcoutsos** [reply]
Sorry, this kept getting buried under other stuff. I finally had a chance to take a proper look and it was as I expected, so the fix is in and it should be formatting labeled nodes in the same manner as unlabeled nodes going forward.

Thanks again!

**Reply by robinovitch61** [reply]
No worries, thanks for hopping on it so quick! I'll update my blog post :)
